app/services/reports/iaa_billing_report.rb

```diff
@@ -57,7 +57,9 @@ def unique_iaa_sps
       sps = []
       ServiceProvider.where.not(
         iaa: nil,
+      ).where.not(
```

Comment:
```
WARNING: NOT conditions will no longer behave as NOR in Rails 6.1. To continue using NOR conditions, NOT each condition individually (`.where.not(:iaa => ...).where.not(:iaa_start_date => ...).where.not(:iaa_end_date => ...)`)
```

`where.not` being NOR is deprecated and will become NAND in 6.1 (https://github.com/rails/rails/issues/31209)

Reply:
Wow that thread was a ride